### PR TITLE
added issue templates for spec

### PR DESCRIPTION
**Pull request by krishchow**
- This PR adds issue templates for the COSI spec - this will be vital as this project progresses so we have a solid framework for tracking issues and requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+**Environment**:
+
+- Kubernetes version (use `kubectl version`), please list client and server:
+- Cloud provider or hardware configuration:
+- OS (e.g: `cat /etc/os-release`):
+- Kernel (e.g. `uname -a`):
+- Install tools:
+- Network plugin and version (if this is a network-related bug):
+- Others:
```
Comment (wlan0) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: Add a line for Provisioner Name and Version

Comment (krishchow) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: Added

**Comment by xing-yang** [comment]
Can you squash your commits?

**Comment by krishchow** [comment]
Squashed them into 1 commit @xing-yang

**Pull request merged**